### PR TITLE
Add image alt text and hover effect

### PR DESCRIPTION
**Pull request by Divinity-dev**
Page title: No issues found
Image text alternatives: Added alternative texts to all images
Text headings: No issues found
Color contrast: No issues found
Resize: No issues found
Interactions: No issues found
Moving content: No issues found
Multimedia: Scaled images to make them larger on hover
The basic structure of the page: No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
             <a href="#About-me" class="icon">About</a>
             <a href="#Contact" class="icon">Contact</a>
             <a href="" class="icon message"><img src="Images/message-16.png" alt=""></a>
-         <img src="Images/532-5328945_menu-bar-icon-white-clipart-png-download-menu.png" alt="" class="dropdown">
+         <img src="Images/532-5328945_menu-bar-icon-white-clipart-png-download-menu.png" alt="Menu bar" class="dropdown">
```
Comment (tufoinnkuo10) [inline index.html:29]: - [ ] kindly use a professional page tittle eg `Mark's portifolio`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Divinity-dev 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
- [ ] kindly add the license(MIT) section in your README file. nice work
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Divinity-dev** [comment]
For the license I didn't add a license when I created the repo.
    On Tuesday, December 20, 2022 at 11:37:02 AM GMT+1, TufoinNkuo ***@***.***> wrote:  
 
 


@tufoinnkuo10 requested changes on this pull request.

Hi @Divinity-dev

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
 ✅ You used the correct Github flow
 ✅ Descriptive pull request message
 ✅ linter error checks validated
 ✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
   
   -  kindly add the license(MIT) section in your README file. nice work   
Optional suggestions   
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In index.html:
> @@ -26,7 +26,7 @@
             <a href="#About-me" class="icon">About</a>
             <a href="#Contact" class="icon">Contact</a>
             <a href="" class="icon message"></a>
-         
+         Menu bar
   
   -  kindly use a professional page tittle eg Mark's portifolio

**Comment by Divinity-dev** [comment]
As for the the page title, the correction has been made

    On Tuesday, December 20, 2022 at 12:01:03 PM GMT+1, Divine Asiriuwa ***@***.***> wrote:  
 
  For the license I didn't add a license when I created the repo.
    On Tuesday, December 20, 2022 at 11:37:02 AM GMT+1, TufoinNkuo ***@***.***> wrote:  
 
 


@tufoinnkuo10 requested changes on this pull request.

Hi @Divinity-dev

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
 ✅ You used the correct Github flow
 ✅ Descriptive pull request message
 ✅ linter error checks validated
 ✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
   
   -  kindly add the license(MIT) section in your README file. nice work   
Optional suggestions   
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In index.html:
> @@ -26,7 +26,7 @@
             <a href="#About-me" class="icon">About</a>
             <a href="#Contact" class="icon">Contact</a>
             <a href="" class="icon message"></a>
-         
+         Menu bar
   
   -  kindly use a professional page tittle eg Mark's portifolio

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @Divinity-dev     👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there!


## Highlights 
:heavy_check_mark: No Linter Errors :+1: 
:heavy_check_mark: Professional PR :+1: 

## Required Changes ♻️
- [ ] As suggested by the previous reviewer **Some of the necessary headers are missing in your Readme 😨**
    - I think it would be better to provide a **LICENSE** link in your **README** as open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 
    _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

## Optional suggestions
**NA**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Divinity-dev** [comment]
But I stated the reason I couldn't add a license to my readme file. I said I didn't add license when I created the repo. How do I add license now?

Sent from Yahoo Mail on Android 
 
  On Tue, Dec 20, 2022 at 12:54 PM, Tuyishimire ***@***.***> wrote:   


@TuyishimireEric requested changes on this pull request.

Status: REQUIRED CHANGES ♻️
 
Hi @Divinity-dev 👋 👋
 
I'm @TuyishimireEric, Your code reviewer,

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there!


Highlights

 ✔️ No Linter Errors 👍
 ✔️ Professional PR 👍
 
Required Changes ♻️
    
   -  As suggested by the previous reviewer Some of the necessary headers are missing in your Readme 😨       
      - I think it would be better to provide a LICENSE link in your README as open source software needs to be licensed to provide rights to modify, distribute other than creator 😂      
we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.


Optional suggestions

NA
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Comment by Divinity-dev** [comment]
Ok. So I just opened the link and it took me to my GitHub account. So if I was to add a file, what's the content to add?

Sent from Yahoo Mail on Android 
 
  On Tue, Dec 20, 2022 at 1:12 PM, Tuyishimire ***@***.***> wrote:   


@Divinity-dev the license file can be added to a repository anytime, it is just a normal file kindy follow this link to add the license to your repository 🙏

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Divinity-dev , 

I am Favour 👩🏽 , your reviewer for this PR.

Great job implementing all the required changes by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**